### PR TITLE
HoverLight Color and Radius are now accessible from code

### PR DESCRIPTION
**Pull request by Alexees**
## Overview
Expands the properties Color and Radius so they can be set via code

## Changes
- Fixes: #7323

### CODE REVIEW

**Comment by cre8ivepark** [comment]
@Cameron-Micka FYI

File: Assets/MixedRealityToolkit/Utilities/StandardShader/HoverLight.cs [hunk 1]
```diff
@@ -26,22 +26,30 @@ public class HoverLight : MonoBehaviour
         /// <summary>
         /// Specifies the Radius of the HoverLight effect
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit/Utilities/StandardShader/HoverLight.cs]: Please make sure to keep the two summary sections next to the public properties.

**Comment by Cameron-Micka** [comment]
Thank you @Alexees!

**Pull request merged**